hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -161,6 +161,10 @@ public void handle(@NonNull final HandleContext handleContext) {
         builder.sequenceNumber(topic.sequenceNumber());
         builder.runningHash(topic.runningHash());
         builder.deleted(topic.deleted());
+        // TODO: added so unit tests pass (fix this when implementing the actual logic)
```

Comment:
This is "actual logic", so should this logic change or should this comment be removed?

Reply:
Removed.
This comment was indication for reviewers, that all changes here fixes unit tests and we still need to add validation logic on these fields in following PRs.